dusty/systems/virtualbox/__init__.py

```diff
@@ -13,10 +13,13 @@
 from ...subprocess import check_and_log_output_and_error_demoted, check_output_demoted, check_call_demoted, call_demoted
 from ...log import log_to_client
 
-def _run_command_on_vm(command_list):
+def _run_command_on_vm(command_list, return_output=False):
     ssh_command = ['docker-machine', 'ssh', constants.VM_MACHINE_NAME]
     ssh_command.extend([command_list])
-    check_and_log_output_and_error_demoted(ssh_command)
+    if return_output:
+        return check_output_demoted(ssh_command)
+    else:
+        check_and_log_output_and_error_demoted(ssh_command)
```

Comment:
Yeah the idea is to toggle both.  The function structure could probably be improved